projects/m1/005-units-of-time-again/python/main.py

```diff
@@ -0,0 +1,13 @@
+#ask to user how many seconds in order to perform computatio
+seconds = int(input("Please insert seconds:\n"))
+
+# divide seconds to obtain the total amount of days, hour, minute, and seconds
+days = seconds // 86400
```

Comment:
It might be a good idea to save the **known values ​​in a constant** and then use it in your code. ;-)
Example: 
`SECONDS_DAY = 86400
 ........ 
 days = seconds // SECONDS_DAY`